### ISSUE TITLE
External storage bug. 

### ISSUE THREAD

**Issue by coraxosm** [owncloud/core#27417]
Hi

ownCloud 8.2

SMB share is opened from Win 2008 R2 domain
Attempt to download a file with properties
"This file came from another computer and might be blocked to help protect this computer" returns only first 26 bytes of file. After unblocking everything works fine.
Very annoying.
Changing user privileges does not solve the problem 

Example
[https://msdnshared.blob.core.windows.net/media/MSDNBlogsFS/prod.evol.blogs.msdn.com/CommunityServer.Components.PostAttachments/00/01/32/31/84/streams_2.jpg](url)

It seems to be a bug in smbclient or in external storage app.

Thanks in advance!

**Comment by kdslkdsaldsal** [comment]
@coraxosm Please fill out the issue template shown when creating a new issue. This is asking you various mandatory info about your environment which are needed to help here (there is a reason why this template is shown).

As you're also mentioning smbclient it worth to deinstall this and use the https://github.com/eduardok/libsmbclient-php which is working more reliable for external storages.

**Comment by coraxosm** [comment]
Thanks a lot, building mentioned extension from source solved the problem.
I guess this issue should be included in some sort of FAQ.
=)

**Comment by kdslkdsaldsal** [comment]
> I guess this issue should be included in some sort of FAQ.

The documentation have some references to this extension so this should be enough from my PoV.

https://doc.owncloud.org/server/latest/admin_manual/configuration_files/external_storage/smb.html
